src/doctest/build.zig

```diff
@@ -99,11 +102,6 @@ pub fn runBuild(
     }
 
     // Add target options
-    // TODO: solve the target mistery and win one less symbol in the lexical scope!
```

Comment:
Admittedly I'm not sure what the mystery referred to in this comment is/was, exactly, but the current logic seems correct to me.